README.md

```diff
@@ -737,6 +737,39 @@ for family in text_string_to_metric_families(u"my_gauge 1.0\n"):
     print("Name: {0} Labels: {1} Value: {2}".format(*sample))
 ```
 
+## Restricted registry
+
+Registry support querying a specific metric with the GET parameter "name[]".
```

Comment:
This is only true if someone is using one of the built in handlers/apps, I wonder if it is worth calling that out? I.e. it won't magically work if someone is calling `generate_latest` from an endpoint themselves.

Reply:
done